### PR TITLE
[batch2] remove /batch endpoint include_jobs param

### PR DESCRIPTION
**Pull request by cseed**
First of a few PRs cleaning up the batch2 API.  Goal is to remove the legacy job spec (now that batch1 is gone) and all endpoints that return multiple items (/batches, /jobs) need to support pagination and search.

### CODE REVIEW

File: batch2/batch/front_end/front_end.py [hunk 1]
```diff
@@ -68,6 +69,36 @@
     return web.Response()
 
 
+async def _get_batch_jobs(app, batch_id, user):
+    db = app['db']
+
+    record = await db.execute_and_fetchone(
+        '''
+SELECT status FROM batches
```
Comment (jigold) [inline batch2/batch/front_end/front_end.py]: I think status is in jobs and not batches

File: hail/python/hailtop/batch_client/aioclient.py [hunk 2]
```diff
@@ -240,12 +230,13 @@ def __init__(self, client, id, attributes):
...
+        batch = await resp.json()
         if include_jobs:
-            params = {'include_jobs': '1'}
-        else:
-            params = None
-        resp = await self._client._get(f'/api/v1alpha/batches/{self.id}', params=params)
-        return await resp.json()
+            resp = await self._client._get(f'/api/v1alpha/batches/{self.id}/jobs')
+            jobs = await resp.json()
+            batch['jobs'] = jobs['jobs']
```
Comment (jigold) [inline hail/python/hailtop/batch_client/aioclient.py]: I think jobs should just be an array based on what you wrote above.

File: hail/python/hailtop/batch_client/aioclient.py [hunk 3]
```diff
@@ -240,12 +230,13 @@ def __init__(self, client, id, attributes):
...
         if include_jobs:
-            params = {'include_jobs': '1'}
-        else:
-            params = None
-        resp = await self._client._get(f'/api/v1alpha/batches/{self.id}', params=params)
-        return await resp.json()
+            resp = await self._client._get(f'/api/v1alpha/batches/{self.id}/jobs')
+            jobs = await resp.json()
+            batch['jobs'] = jobs
+        reutrn batch
```
Comment (jigold) [inline hail/python/hailtop/batch_client/aioclient.py]: return is misspelled

**Comment by cseed** [comment]
OK, this should be fixed now.

File: batch2/batch/batch.py [hunk 4]
```diff
@@ -14,7 +14,7 @@
 log = logging.getLogger('batch')
 
 
-async def batch_record_to_dict(db, record, include_jobs=False):
+async def batch_record_to_dict(record):
```
Comment (jigold) [inline batch2/batch/batch.py]: This doesn't need to be async anymore

**Pull request merged**